lib/event_tracker.rb

```diff
@@ -0,0 +1,55 @@
+require './event_tracker_haystack.rb'
+
+divider
```

Comment:
naming suggstion `print_divider`.  this would be a little bit clearer on what it does